tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -86,29 +85,11 @@ def format_message(self, alert):
         return "Subject: [Alert] %s, %s threshold breached\n\n%s" % (
             alert.resource, alert.severity, alert.tags['message'])
 
-    def get_alert_destinations(self, alert):
```

Comment:
But now, you'll probably have something like a global index of user to subscription kind of a thing that's what @anivargi and @brainfunked were saying right.. So where are you planning to iterate that list then??